cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -1044,14 +1048,43 @@ func WatchConfigMap() {
 				} else {
 					glog.V(1).Infof("Logging level is changed from %v to %v", currentLoggingLevel, newLoggingLevel)
 				}
-
 			}
 		}
+
```

Comment:
I don't think you need to pass `viper.GetString`  here, why don't you call it directly in the function `updateNodePoolConfig`？

Reply:
I see your point